common/special_troops/wc_special_troops.txt

```diff
@@ -118,62 +118,62 @@ demonicshock = {
 	base_type = heavy_infantry
 }
 
-artillery = {
-   morale = 5
-   maintenance = 15
-   phase_skirmish_attack = 15
+siege_machines = {
+   morale = 50
+   maintenance = 10
+   phase_skirmish_attack = 10
    phase_melee_attack = 0.25
-   phase_pursue_attack = 4
-   phase_skirmish_defense = 15
+   phase_pursue_attack = 2
+   phase_skirmish_defense = 10
    phase_melee_defense = 0.25
-   phase_pursue_defense = 0
+   phase_pursue_defense = 2
    base_type = archers
 }
 
-dragon_fletchlings = {
+thralls = {
 	morale = 5
-	maintenance = 1
-	phase_skirmish_attack = 3
+	maintenance = 0.3
+	phase_skirmish_attack = 1
 	phase_melee_attack = 1
-	phase_pursue_attack = 2
-	phase_skirmish_defense = 2
+	phase_pursue_attack = 1
+	phase_skirmish_defense = 1
 	phase_melee_defense = 1
-	phase_pursue_defense = 2
-	base_type = archers
+	phase_pursue_defense = 1
+	base_type = heavy_infantry
 }
 
 dragons = {
-	morale = 15
-	maintenance = 25
-	phase_skirmish_attack = 25
-	phase_melee_attack = 25
-	phase_pursue_attack = 10
-	phase_skirmish_defense = 15
-	phase_melee_defense = 15
-	phase_pursue_defense = 15
-	base_type = archers
+	morale = 40
+	maintenance = 12
+	phase_skirmish_attack = 8
+	phase_melee_attack = 8
+	phase_pursue_attack = 18
+	phase_skirmish_defense = 8
+	phase_melee_defense = 8
+	phase_pursue_defense = 18
+	base_type = heavy_cavalry
```

Comment:
Large Infantry, not Aerial?

Reply:
I was stuck thinking Dragon enemies in Warcraft attack far more often with claws and biting, but I suppose that makes more sense in this context